src/script/config/dependenciesGraph.js

```diff
@@ -81,7 +79,6 @@ export {
   BackendClient,
   BackupService,
   BroadcastService,
-  CacheRepository,
```

Comment:
Why did you remove this?

Reply:
Because it has static functions now and does not need to be resolved anymore.